docs/resources/rule_parser_state.md

```diff
@@ -29,53 +29,53 @@ resource "wallarm_rule_parser_state" "disable_xml_parsing" {
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
-* `parser` - (Required) Parser to enable/disable. Possible values: `base64`, `cookie`, `form_urlencoded`, `gzip`, `grpc`, `json_doc`, `multipart`, `percent`, `protobuf`, `htmljs`, `viewstate`, `xml`.
-* `state` - (Required) Desired state for the parser. Possible values: `enabled`, `disabled`.
-* `point` - (Required) Request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `client_id` - (optional) ID of the client to apply the rules to. The value is required for [multi-tenant scenarios][2].
+* `action` - (optional) rule conditions. Possible attributes are described below.
+* `parser` - (**required**) parser to enable/disable. Possible values: `base64`, `cookie`, `form_urlencoded`, `gzip`, `grpc`, `json_doc`, `multipart`, `percent`, `protobuf`, `htmljs`, `viewstate`, `xml`.
+* `state` - (**required**) desired state for the parser. Possible values: `enabled`, `disabled`.
+* `point` - (**required**) request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
 
 **action**
 
 `action` argument shares the available
 conditions which can be applied. The conditions are:
 
-* `type` - (Optional) Condition type. Can be: `equal`, `iequal`, `regex`, `absent`. Must be omitted for the `instance` parameter in `point`.
+* `type` - (optional) condition type. Can be: `equal`, `iequal`, `regex`, `absent`. Must be omitted for the `instance` parameter in `point`.
   For more details, see the offical [Wallarm documentation](https://docs.wallarm.com/user-guides/rules/add-rule/#condition-types)
   Example:
   `type = "absent"`
-* `value` - (Optional) Value of the parameter to match with. Must be omitted for the `instance` parameter in `point` or if `type` is `absent`.
+* `value` - (optional) value of the parameter to match with. Must be omitted for the `instance` parameter in `point` or if `type` is `absent`.
   Example:
   `value = "example.com"`
-* `point` - (Optional) Request parameters that trigger the rule. Possible values are described below. For more details, see the official [Wallarm documentatioon](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `point` - (optional) request parameters that trigger the rule. Possible values are described below. For more details, see the official [Wallarm documentatioon](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
 
 **point**
 
-  * `header` - (Optional) Arbitrary HEADER parameter name.
```

Comment:
there are more points in UI, check all docs

Reply:
Compating to [this](https://docs.wallarm.com/user-guides/rules/add-rule/#points) list:

- `query` is missing in terraform doc
- in terraform doc we have `instance`, in the docs it is `application`

@aoleynikov-wallarm, is `query` supported? Is `instance` the right name?

Reply:
`query` is not supported currently. We should probably create tickets for all such missing features.

`instance` is still the right name on the API level, yes.